src/com/mellanox/cicd/Matrix.groovy

```diff
@@ -63,6 +63,8 @@ def forceCleanupWS() {
 }
 
 def gen_image_map(config) {
+    //def supported_arch_list = ['x86_64', 'aarch64', 'ppc64le']
+    def supported_arch_list = ['x86_64', 'aarch64']
```

Comment:
should not be hardcoded, it should load config.kubernetes.arch_table keys and use is as supported arch list

Reply:
Fixed.